crates/runtime/src/starknet/resources/versioned_constants_13_0.json

```diff
@@ -0,0 +1,394 @@
+{
```

Comment:
We have to copy this since we can't just load it from the crate, and defaults for 0.5.0 will be ones for 0.13.1 starknet

Reply:
I guess now we can load the defaults xD

Reply:
If we want to merge it after Wednesday won't we want to migrate to 0.13.1 asap anyways? I would do it when everything else in this PR is settled, as a last step, but that's up 2 u

Reply:
yes that makes sense, but i'd rather leave it for the next PR since it incurs changes that will not be comfortable to easily verify here